AzureResourceManager/azuredeploy.json

```diff
@@ -0,0 +1,864 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vmSku": {
+      "type": "string",
+      "defaultValue": "Standard_A4",
+      "metadata": {
+        "description": "Size of VMs in the VM Scale Set."
+      }
+    },
+    "clusterName": {
+      "type": "string",
+      "metadata": {
+        "description": "String used as a base for naming resources. Must be 3-61 characters in length and globally unique across Azure. A hash is prepended to this string for some resources, and resource-specific information is appended."
+      },
+      "maxLength": 61
+    },
+    "nodeCount": {
+      "type": "int",
+      "defaultValue": 2,
+      "minValue": 1,
+      "metadata": {
+        "description": "Number of VM instances (100 or less) for member artifactory nodes. Minimum 1 node."
+      },
+      "maxValue": 100
+    },
+    "artifactoryVersion": {
+      "type": "string",
+      "defaultValue": "5.8.1",
+      "metadata": {
+        "description": "Artifactory version to deploy. Supported from 5.8.0 and above."
+      }
+    },
+    "masterKey": {
+      "type": "string",
+      "defaultValue": "1ce2be4490ca2c662cb79636cf9b7b8e",
+      "metadata": {
+        "description": "Master key for Artifactory cluster. Generate master.key using command '$openssl rand -hex 16'"
+      },
+      "maxLength": 64
+    },
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "Admin username on all VMs. Follow conventions for azure VM admin user name."
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Admin password on all VMs. Follow conventions for azure VM admin password rules."
+      }
+    },
+    "ubuntuOSVersion": {
+      "type": "string",
+      "defaultValue": "14.04.2-LTS",
```

Comment:
I'm pretty sure the default ubuntu should be xenial, not trusty, which is 16.04 . . . I think Azure supports that?